integration_tests/google/type/date.proto

```
@@ -0,0 +1,54 @@
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.type;
```

Buf (1.67.0)

[error] Files with package "google.type" must be within a directory "google/type" relative to root but were in directory "integration_tests/google/type".

(PACKAGE_DIRECTORY_MATCH)